### DISCUSSION TITLE
[Foodtype Suggestion] Curry

### DISCUSSION THREAD

**Discussion by dhi-holo** [vectorwing/FarmersDelight#533] [Ideas]
While messing about with the mod (specifically the fabric port, but that's not exactly relevant for this post) I noticed that Rice only has a few uses, and with the Tatami blocks also added by the mod, I thought some types of curry could be interesting to see.

Utilising rice, chicken cuts, milk, onions or tomato, and perhaps other ingredients, a bowl of curry could be created. Perhaps some alternate types of curries, with different sauce ingredients and meats could also exist, similar to stews/soups already present in the mod. The other ingredients could include stuff like cocoa beans, mutton chops, fish slices, bacon, or eggs.

**Comment by RaymondBlaze** [comment]
For the curry's spicy flavor, maybe we could add blaze powder or magma cream as a ingredient :P
